travis/pull_request_setup.bash

```diff
@@ -0,0 +1 @@
+#!/bin/bash
```

Comment:
At this point, I don't think there's anything special needed for PR builds, so we can probably drop the file